config/routes.rb

```diff
@@ -16,18 +19,36 @@
   delete '/cart', to: 'carts#delete_item', as: 'cart'
   put '/cart', to: 'carts#update_item_quantity', as: 'update_item_quantity'
 
-  resources :items
-
   namespace :admin do
     get '/dashboard', to: 'base#dashboard'
-    resources :categories
-    resources :items
-    resources :orders
+    # resources :categories
+    # resources :items
+    # resources :orders
   end
 
   resources :users do
     resources :orders
   end
 
+  # resources :tenants, path: "", param: :tenant
+
+  scope ':tenant', module: 'tenants' do
+    get '/' => 'items#index', as: :tenant
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.